tests/landscape.sil

```diff
@@ -0,0 +1,8 @@
+\document[orientation=landscape,papersize=a5]{
+%\use[module=packages.frametricks]
+%\showframe[id=all]
+
+\use[module=packages.lorem]
+\lorem[words=1000]
+
+}
```

Comment:
Why not `\begin ... \end{document}` (more usual way to way to wrap a document!)

By the way, for tests, when sure it does the right thing, you need to run `sile tests/landscape.sil -b debug` to generate a `.debug` text file, rename it as `.expected` and commit it too. The test checks (and the CI) use this for regression testing.

Reply:
Ok.
N.B. Good practice = Let the reviewer resolve the conversation after they have a chance to check.

Reply:
For updating expectations, you can just run `make tests/landscape.expected` once you are getting the results you actually do expect.